app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    check_list = command.split(" ")
+    if check_list[0] != "cp" or check_list[1] == check_list[2]:
```

Comment:
The condition `check_list[1] == check_list[2]` is intended to prevent copying a file onto itself. However, this check will raise an `IndexError` if the command does not contain at least three parts. Consider adding a check to ensure `check_list` has the required number of elements before accessing them.